optaplanner-examples/src/test/resources/org/optaplanner/examples/integrationtests/lowentitycount/data/oneDomicileOneEntity.xml

```diff
@@ -0,0 +1,28 @@
+<TspSolution id="1">
```

Comment:
These xml files pose a maintenance issue. Every time we refactor an example's model, they break. And I refactor example models regularly.

Yes, we have similar xml files in the unsolved directory - but those get regenerated with "Generator" or "Importer" every time an examples model changes. That's little work.

Yes, we have a few solved or initialized files in some example, maybe 3 in total. And yes, those are a pita and I want to get rid of them.

So let's not add any test xml's if we have a different approach:
Because each of these models are used in exactly one test, create that problem instance on the fly in that test.
See https://github.com/kiegroup/optaplanner/blob/master/optaplanner-examples/src/test/java/org/optaplanner/examples/conferencescheduling/solver/ConferenceSchedulingScoreHardConstraintTest.java#L62 for an example.
Notice it makes heavy use of "with*()" builder methods. Feel free to add any on the TSP domain, similar to how it's done here: https://github.com/kiegroup/optaplanner/blob/master/optaplanner-examples/src/main/java/org/optaplanner/examples/conferencescheduling/domain/Timeslot.java#L148

Reply:
@ge0ffrey Ok, now I see the point - it's not just about the big datasets, but rather about the maintenance of these files in general, so we want to get rid of all of them.